frontend/src/metabase-lib/expressions/__support__/expressions.ts

```diff
@@ -107,19 +109,20 @@ const database = createMockDatabase({
     "foreign-keys",
     "native-parameters",
     "expressions",
-    "advanced-math-expressions",
+    // TODO: uladzimir. Either fix a type of remove this feature
+    // Type '"advanced-math-expressions"' is not assignable to type 'DatabaseFeature'
+    // "advanced-math-expressions",
```

Comment:
I think we should update `DatabaseFeature`

Reply:
fixed https://github.com/metabase/metabase/pull/36265/commits/409d772b352acc598cb8e3372d555ab73dee9fc5